lib/wibox/init.lua

```diff
@@ -384,6 +384,10 @@ local function new(args)
         ret.border_color = args.border_color
     end
 
+    if args.opacity then
```

Comment:
I *think* adding `opacity` to the `force_forward` set will be enough. The code around this could probably be refactored to use `force_forward` too.

Reply:
I'm not sure I understand you correctly but just adding `opacity = true` to `force_forward` does not make a difference, the `__newindex` metatable does not run unless we set `ret.opacity = args.opacity` when using the declarative syntax. The imperative one works either way.